middleware/domain/source/manager/handle.cpp

```diff
@@ -908,6 +908,34 @@ namespace casual
                   };
                }
 
+               namespace information
+               {
+                  auto request( State& state)
+                  {
+                     return [&state]( const common::message::domain::process::information::Request& message)
+                     {
+                        Trace trace{ "domain::manager::handle::process::information::request"};
+
+                        auto reply = common::message::reverse::type( message);
+                        algorithm::for_each( message.handles, [&state, &reply]( const auto& handle)
+                        {
+                           if( handle.pid)
+                           {
+                              if( auto server = state.server( handle.pid))
+                                 reply.processes.emplace_back( server->instance( handle.pid)->handle, server->alias, server->path);
+                              else if( auto executable = state.executable( handle.pid))
+                                 // we make sure to return a handle with pid only, since executables lack ipc
+                                 reply.processes.emplace_back( common::process::Handle{ handle.pid}, executable->alias, executable->path);
```

Comment:
Bit unsure about this. But it makes some amount of sense that one should be able to get information about executables as well.